.github/workflows/Perf_Env_Build_Test_CI.yml

```diff
@@ -302,6 +302,8 @@ jobs:
           python -m pip install --upgrade pip
           pip install benchmark-runner wheel
       - name: 🎁 Bump Version
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
not sure if you need to add to the checkout as well like: 

https://github.com/redhat-chaos/actions/blob/8f9fc488175d7135618259b40385e6ddd321db8a/prow-scripts/action.yml#L19-L23